stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -489,6 +487,50 @@
         <item name="streamUiSearchInputViewTextSize">@dimen/stream_ui_text_medium</item>
     </style>
 
+    <style name="StreamUi.AttachmentGallery" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+        <item name="streamUiAttachmentGalleryCloseButtonStyle">@style/StreamUi.AttachmentGallery.CloseButton</item>
+        <item name="streamUiAttachmentGalleryTitleStyle">@style/StreamUi.AttachmentGallery.Title</item>
+        <item name="streamUiAttachmentGalleryDateStyle">@style/StreamUi.AttachmentGallery.Date</item>
+        <item name="streamUiAttachmentGalleryActionsMenuStyle">@style/StreamUi.AttachmentGallery.ActionsMenu</item>
+        <item name="streamUiAttachmentGalleryBottomBarImageCountStyle">@style/StreamUi.AttachmentGallery.BottomBar.ImageCount</item>
+        <item name="streamUiAttachmentGalleryBottomBarLeftIconStyle">@style/StreamUi.AttachmentGallery.BottomBar.LeftIcon</item>
+        <item name="streamUiAttachmentGalleryBottomBarRightIconStyle">@style/StreamUi.AttachmentGallery.BottomBar.RightIcon</item>
+    </style>
+
+    <style name="StreamUi.AttachmentGallery.CloseButton" parent="StreamUi">
+        <item name="android:src">@drawable/stream_ui_ic_close</item>
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_marginStart">@dimen/stream_ui_spacing_small</item>
+    </style>
+
+    <style name="StreamUi.AttachmentGallery.Title" parent="StreamUi">
+        <item name="android:textAppearance">@style/StreamUiTextAppearance.HeadlineBold</item>
```

Comment:
We are using our own `TextStyle` in other places - check `StreamUi.SuggestionListView` for example. Shouldn't we do the same in this Activity?

Reply:
That's a complex one. Discussing with the team...